src/support/notfound.js

```diff
@@ -15,6 +15,7 @@ import commaNumber from 'comma-number';
 import { markdown, section } from './slack.js';
 
 export const INITIAL_404_SLACK_MESSAGE = '*404 REPORT* for the *last week* :thread:';
+export const NO_404_SLACK_MESSAGE = 'Good news ! No 404 error was detected';
```

Comment:
Extra space between `Good news` and `!`

maybe the message could be re-written to be alike with the initial one:

```
Good news! for ${url}, there were no 404 errors last week for the end users
```